draft-ietf-quic-ack-frequency.md

```diff
@@ -382,15 +382,20 @@ Unreported Missing:
   Missing that have not yet been received.
 
 An endpoint that receives an ACK_FREQUENCY frame with a non-zero Reordering
-Threshold value SHOULD send an immediate ACK when the gap
-between the smallest Unreported Missing packet and the Largest Unacked is greater
-than or equal to the Reordering Threshold value. Sending this additional ACK will
-reset the max_ack_delay timer and Ack-Eliciting Threshold counter (as any ACK
-would do).
-
-When an ack-eliciting packet is received with a packet number less than
-Largest Acked, this still triggers an immediate acknowledgement in an effort to
-avoid the packet being spuriously declared lost.
+Threshold value SHOULD send an immediate ACK whenever it receives an ack-eliciting, out-of order packet
+whose packet number is outside the reordering window of the peer, i.e. when
+* the gap between the smallest Unreported Missing packet and the
```

Comment:
```suggestion
* the difference between the smallest Unreported Missing packet and the
```

As to @MarcoPolo's point, I think we can actually change "gap" to "difference" in this pull request and call it a day.

The word "gap" is used three times in the current draft, but this is the only case where the word is used as a synonym of "difference." The other two cases use the word to indicate a hole in the packet numbers being received.